app/bundles/FormBundle/Form/Type/FormFieldSliderType.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Mautic\FormBundle\Form\Type;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\IntegerType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+/**
+ * @extends AbstractType<FormFieldSliderType>
+ */
+class FormFieldSliderType extends AbstractType
```

Comment:
Make all new classes final unless there is a reason not to.
```suggestion
final class FormFieldSliderType extends AbstractType
```